common/app/model/meta/LinkedData.scala

```diff
@@ -29,8 +38,8 @@ object Logo {
 }
 
 case class Guardian(
-  override val `@type`: String = "Organization",
-  override val `@context`: String = "http://schema.org",
+  `@type`: String = "Organization",
+  `@context`: String = "http://schema.org",
```

Comment:
minor but `https` 🔒 (few in the PR)

Reply:
I had a look at this and am a bit unsure if/how this matters. All of the examples I can find (including on schema.org) use the non-https version so was thinking of sticking with that.

Reply:
Update: actually, the Google examples do use https and I agree this seems logical! Have changed.